components/eventing-controller/controllers/subscription/jetstream/reconciler_test.go

```diff
@@ -468,6 +468,36 @@ func TestChangeSubscription(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "CleanEventTypes; update valid filter to a filter with an invalid prefix",
+			givenSubscriptionOpts: []reconcilertesting.SubscriptionOpt{
+				reconcilertesting.WithFilter(reconcilertesting.EventSource, utils.NewCleanEventType("0")),
+				reconcilertesting.WithWebhookForNATS(),
+				reconcilertesting.WithSinkURLFromSvc(ens.SubscriberSvc),
+			},
+			wantBefore: utils.Want{
+				K8sSubscription: []gomegatypes.GomegaMatcher{
+					reconcilertesting.HaveCondition(reconcilertesting.DefaultReadyCondition()),
+					reconcilertesting.HaveSubsConfiguration(utils.ConfigDefault(ens.DefaultSubscriptionConfig.MaxInFlightMessages)),
```

Comment:
Why do we check the `Subscription.Status.Config` property ? It is required for the test ?

Reply:
Fair point! It's not required for this test. Will remove it

Reply:
Done